Extension/src/LanguageServer/configurations.ts

```diff
@@ -1896,7 +1896,6 @@ export class CppProperties {
         }
 
         // Resolve and split any environment variables
-        paths = this.resolveAndSplit(paths, undefined, this.ExtendedEnvironment);
         compilerPath = util.resolveVariables(compilerPath, this.ExtendedEnvironment).trim();
```

Comment:
It may also be possible to remove the call to `util.resolveVariables` since I think it is done in `this.resolvePath`.  You'd need to keep the `trim` though.

Reply:
That's a good point, it does look like the call to resolveVariables is reundant.